### ISSUE TITLE
When installing packages from (win) explorer, main window of playnite is not brought to front every time

### ISSUE THREAD

**Issue by joyrider3774** [JosefNemec/Playnite#2071]
**Check for existing issue**
nothing found

**Describe the bug**
When you install extensions and plugins using packages from explorer.exe the first time you try to install one after starting playnite, the main playnite window is not brought to front and you have to click the icon in the taskbar to actually activate playnite again and see the message about installing the extension plugin or them. If you cancel the package installation by choosing no and retry all the steps it starts working, thats why i think the problem happens only the 1st time after starting playnite. If you downloaded many themes at once and you want to install them all this might get tedious as after each install of for example a theme a restart is required and the problem happens again upon installing the next package

**To Reproduce**
1. download a package (for example a theme)
2. start playnite and maximize the window
3. start explorer.exe maximize as well (although not really needed) goto folder containing the package and double click
4. playnite has a message for installing the extension but main window is not brought to front 

**Screenshots**
see youtube video here https://youtu.be/CI6bB0uJIzI

**Diagnostics ID**
none

**Comment by JosefNemec** [comment]
Is this still an issue with the latest version?

**Comment by joyrider3774** [comment]
Yes still happens in current / latest version. Main playnite window is not brought to front, have to manually click it in taskbar
